### ISSUE TITLE
Input focus lost when updating "isResizable" or "isDraggable"

### ISSUE THREAD

**Issue by sopretty** [STRML/react-grid-layout#1028]
- [x] Bug
- [ ] Feature request
- [ ] Question

Note: I can only reproduce this issue on Chrome.

In my example, I want to set the `isResizable` or `isDraggable` props to false once I start typing on a text input. But when the values change, I can't keep typing anymore in the input, I have to focus the input again.

Here's a sandbox example (remember to try on Chrome)
https://codesandbox.io/s/pedantic-faraday-cedgd

**Steps to reproduce:**
1. The grid items are resizable and draggable
2. Focus input
3. Enter one letter
4. The grid items are not resizable and draggable | The focus of the input is lost

**Comment by ElaiNili** [comment]
I also had the same problem, I resolved it in this way. 

```
static get defaultProps() {
    return {
      draggableCancel: '.nondrag',
    };
  }
```
_______________
```
<ResponsiveReactGridLayout
          { ...this.props} >
```
______________
`<div className="nondrag">`

NOTICE: do not forget to pass props into ResponsiveReactGridLayout!

**Comment by palashkaria** [comment]
I'm facing the same issue when using an input to filter draggable list. The fix provided in #1113 worked in my case; hoping to see a release soon!
